docs/releases/1.9.txt

```diff
@@ -528,8 +528,14 @@ Templates
 * Added a :meth:`Context.setdefault() <django.template.Context.setdefault>`
   method.
 
-* A warning will now be logged for missing context variables. These messages
-  will be logged to the :ref:`django.template <django-template-logger>` logger.
+* A DEBUG level message will now be logged for missing context variables.
```

Comment:
Maybe we can restructure this and say something like: "The new django.template logger logs the following:" (with sub-bullets).